zerver/openapi/zulip.yaml

```diff
@@ -3197,8 +3197,20 @@ paths:
                                 realm_linkifiers:
                                   type: array
                                   description: |
-                                    Array of dictionaries where each dictionary contains details about
-                                    a single realm linkifier.
+                                    An ordered array of dictionaries where each
+                                    dictionary contains details about a single
+                                    realm linkifier.
+
+                                    The order of the array reflects the order
+                                    that each linkifier is applied when
+                                    linkifying messages and topics. By default,
+                                    new linkifiers are ordered last. This order
+                                    can be modified with [`PATCH
+                                    /realm/linkifiers`](/api/reorder-linkifiers).
+
+                                    **Changes**: Changed in Zulip 8.0 (feature level 201).
+                                    Linkifiers became consistently ordered with the order
+                                    that they would be applied.
```

Comment:
I think this can be simplified to just extend the previous label with the detail that linkifiers are sent in the order in which they should be processed, and the order can be modified with `[`PATCH /realm/linkifiers`](/api/reorder-linkifiers)`, with no changes note.

(I say "should" above because clients doing local echo need to understand this in order to themselves do local echo)

Reply:
Removed the changes section and change to wording to use `should` instead of `is`.